app/components/primer/auto_complete.rb

```diff
@@ -5,15 +5,21 @@ module Primer
   class AutoComplete < Primer::Component
     status :beta
 
+    DEFAULT_TAG = :"auto-complete"
+
     DEFAULT_INPUT_TYPE = :text
     INPUT_TYPE_OPTIONS = [DEFAULT_INPUT_TYPE, :search].freeze
+    DEFAULT_INPUT_TAG = :input
+
+    DEFAULT_RESULTS_TAG = :ul
 
     # Required input used to search for results
     #
-    # @param type [Symbol] <%= one_of(Primer::AutoComplete::INPUT_TYPE_OPTIONS) %>
+    # @param type [Symbol] (<%= pretty_value(Primer::AutoComplete::DEFAULT_INPUT_TYPE) %>) <%= one_of(Primer::AutoComplete::INPUT_TYPE_OPTIONS) %>
+    # @param tag [Symbol] (<%= pretty_value(Primer::AutoComplete::DEFAULT_INPUT_TAG) %>) <%= locked_tag_text %>
```

Comment:
Feels a little cluttered having to call `pretty_value` but need in order to retain symbol as string.